examples/finding_sunspots_using_stara.py

```diff
@@ -0,0 +1,99 @@
+"""
+============================
+Finding sunspots using STARA
+============================
+
+This example demonstrates the use of Sunspot Tracking And Recognition
+Algorithm (STARA) in detecting and plotting sunspots. More information
+on the algorithm can be found in [this](https://arxiv.org/abs/1009.5884) paper.
```

Comment:
Is this paper not published now? 

If so, we should replace this URL with a DOI URL.

Reply:
okay done.